projects/pydentity-wallet.md

```diff
@@ -0,0 +1,58 @@
+# Project Name
+PyDentity Wallet
+
+# Preferred Maturity Level
+Labs stage
+
+# Project Description
+The PyDentity Wallet is a Progressive Web Application for providing [acapy](https://github.com/openwallet-foundation/acapy) access to mobile wallet features and browser API access.
+
+The wallet provides a frontend for a multitenant acapy instance which can be bound to a local mobile device storage.
+
+The app works with iOS and Android and allows users to easily manage an acapy subwallet.
+
+Whenver possible, it uses existing open standards from W3C, IETF, DIF, and other open standards groups.
+
+# Alignment with the OpenWallet Foundation Mission
+This project aligns with the OWF mission, because it's an open source implementation of wallet-related open standards and enhances an existing OWF impact project.
+
+# Code of Conduct
+[OpenWallet Foundation code of conduct](https://tac.openwallet.foundation/governance/code-of-conduct/)
+
+# TAC Sponsor
+[Stephen Curran](https://github.com/swcurran)
+
+# Project License
+Apache 2.0
+
+# Source Control
+Currently on a private repository, this project will be a new project at OWF and all existing code will be contributed.
+
+# Issue Tracker
+N/A (not applicable), will track issues through the new GH project.
+
+# External Dependencies
+[Tabler](https://github.com/tabler/tabler) ([MIT](https://github.com/tabler/tabler/blob/dev/LICENSE))
+
+Open Sourced Python Packages, managed with poetry (see pyproject.toml)
```

Comment:
I guess this `pyproject.toml` is in the private repository

Reply:
It's good to have a full license dump to confirm that we do not have dependencies on non-permissive licenses.

Reply:
I will be happy to join a [SBOM](https://www.cisa.gov/sbom) with the initial PR along with license info